### DISCUSSION TITLE
Sémiologie - icone image

### DISCUSSION THREAD

**Discussion by opetitprestataire** [IGNF-Ma-carte/.github#9] [Visualiseur]
Bonjour ! J'ai importé une icone avec la sémiologie pour illustrer mes ponctuels (mode image) : 
![icone](https://github.com/user-attachments/assets/f50279f9-f493-416e-8809-a8aaf6edca80)
Or ceux ci ne sont pas persistés dans la carte en consultation 
https://macarte.ign.fr/carte/nSgew8/Carte-des-implantations-de-l-OFB

Qu'ai-je mal fait ?
Merci beaucoup

**Comment by Viglino** [comment]
Chez moi ça a l'air de fonctionner 
![Screenshot_20241222_151616_Chrome](https://github.com/user-attachments/assets/ab92cdc7-7336-4a72-9b18-75bad9d345bb)
